src/main/java/strings/StringManipulation.java

```diff
@@ -0,0 +1,20 @@
+package strings;
+
+public class StringManipulation {
+
+    public String reverse(String str) {
+        StringBuilder builder = new StringBuilder();
+        for (int i=str.length()-1;i>=0;i--) {
+            builder.append(str.charAt(i));
+        }
+        return builder.toString();
+    }
+    
+    public boolean isPlanindrome(String str) {
+        if (str.equals(reverse(str))) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this if-then-else statement by a single return statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ec2-54-187-222-128.us-west-2.compute.amazonaws.com:9000/sonar/coding_rules#rule_key=squid%3AS1126)